libparsec/crates/client/tests/unit/certif/ensure_realm_created.rs

```diff
@@ -0,0 +1,42 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_tests_fixtures::prelude::*;
+use libparsec_types::prelude::*;
+
+use crate::certif::{CertifEnsureRealmCreatedError, CertificateBasedActionOutcome};
+
+use super::utils::certificates_ops_factory;
+
+#[parsec_test(testbed = "minimal")]
+async fn ok(env: &TestbedEnv) {
+    let (env, realm_id) =
+        env.customize_with_map(|builder| builder.new_realm("alice").map(|event| event.realm_id));
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    ops.add_certificates_batch(
+        &env.get_common_certificates_signed(),
+        &[],
+        &[],
+        &env.get_realms_certificates_signed(),
+    )
+    .await
+    .unwrap();
+
+    let res = ops.ensure_realm_created(realm_id).await.unwrap();
+
+    p_assert_matches!(res, CertificateBasedActionOutcome::LocalIdempotent);
+}
+
+#[parsec_test(testbed = "minimal")]
+async fn empty(env: &TestbedEnv) {
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    let err = ops
+        .ensure_realm_created(VlobID::default())
+        .await
+        .unwrap_err();
+
+    p_assert_matches!(err, CertifEnsureRealmCreatedError::Offline);
```

Comment:
I'm not sure about the test name, currently it's `empty` but the error value is `offline`

Reply:
Let's call it offline then